snuba/datasets/storage.py

```diff
@@ -31,16 +31,10 @@ class Storage(ABC):
     for more useful abstractions.
     """
 
-    def __init__(
-        self, storage_key: StorageKey, storage_set_key: StorageSetKey, schema: Schema
-    ):
-        self.__storage_key = storage_key
+    def __init__(self, storage_set_key: StorageSetKey, schema: Schema):
         self.__storage_set_key = storage_set_key
         self.__schema = schema
 
-    def get_storage_key(self) -> StorageKey:
-        return self.__storage_key
-
```

Comment:
Why this change? It does not seem related to joins

Reply:
I moved it to line 112 since only table storages have a storage key now. I could leave it here and return None for joined storages.